### PR TITLE
FUCK scalerot

### PR DESCRIPTION
**Pull request by LanCartwright**
remove liggers

also scalerot

:cl: 
add: Scale rot.
add: Scale rot to Mining vendors.
add: Scale rot to disease outbreak event.
add: Scale rot medipens.
add: Scale rot reagents.
add: Scale rot to cargo ordering.
add: Scale rot cure/vaccine autoinjectors.
/:cl:

### CODE REVIEW

**Comment by InsaneHyena** [comment]
Now this is a PR I can get behind

**Comment by ShizCalev** [comment]
You're using the old paths. Everything that was /obj/item/weapon is now just /obj/item

File: code/datums/diseases/scalerot.dm [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+	severity = DANGEROUS
+	longevity = 650
+
+/datum/disease/scalerot/stage_act()
+	..()
+	if(affected_mob.dna.species.id == "lizard"||affected_mob.dna.species.id == "fly") //Non-scaled species are not affected by the virus and merely carry it.
+		switch(stage)
+			if(1)
+				if(prob(40))
+					if(affected_mob.dna.species.id == "lizard")
```
Comment (DoctoreMad) [inline code/datums/diseases/scalerot.dm]: Still doesn't include the actual race ash lizards have, which is different, so they'll get no comment that something is wrong.

**Comment by calzilla1** [comment]
Yus

File: code/datums/diseases/scalerot.dm [hunk 2]
```diff
@@ -0,0 +1,95 @@
+/datum/disease/scalerot
+	name = "Scale rot"
+	max_stages = 5
+	spread_text = "On contact"
+	spread_flags = CONTACT_GENERAL
+	cure_text = "Alcohol based anti-septics or vitamin superdoses."
```
Comment (Fox-McCloud) [inline code/datums/diseases/scalerot.dm]: This is wrong.

**Comment by Kromgar** [comment]
The liggers need to learn their place

**Comment by Repukan** [comment]
Deus vult

**Comment by Armhulen** [comment]
YES

**Comment by kevinz000** [comment]
even a permaban can't stop some people from ruining the server.

**Comment by Armhulen** [comment]
why do flies get the rear end of this? It just seems kinda out of place, they're not reptilian at all.

**Comment by FantasticFwoosh** [comment]
I wouldn't have spread it so far and wide given its a grievious disease that only affects lizards and flypeople into every conceivable place you can.

- Traitor uplink bottle of scalerot for traitor virologists for like 2 or three TC would have been fine, as well as just adding scalerot to some culture bottle imports.

We're going to get complaints where people are going to die as a result of non-antags releasing this because "**_race war now!_**" escalated hostility memes rather than actual presentable threats.
_________________________________________________________
**Make it rot off catpeople tails & ears!**

**Comment by Hatfish** [comment]
where are the fleas you cat-loving memelord

**Comment by XDTM** [comment]
Time to add more snowflake races to make up for this

**Comment by FantasticFwoosh** [comment]
> Time to add more snowflake races to make up for this 

🤡 Honk! 🤡

**Comment by pubby** [comment]
I think people are downvoting because they see it as JMad grudging against lizards.

But honestly we could use more diseases in the game, and species-specific diseases are both logical and flavorful. You can't improve medical without breaking a few eggs.

**Comment by BeeSting12** [comment]
@pubby 
Adding it to the mining vendor is my main issue. It shouldn't be that easy to access a huge grief tool for lizards.

**Comment by ExcessiveUseOfCobblestone** [comment]
Time to unban jmad already?

**Comment by LanCartwright** [comment]
ill fix it tonight

**Comment by Shadowlight213** [comment]
:+1:

**Comment by kevinz000** [comment]
jmad give everyone a racewar disease or no one reee

**Comment by calzilla1** [comment]
"Give every race a language or no one reee" shut up kev

**Comment by kevinz000** [comment]
never

**Comment by LanCartwright** [comment]
alright will add disease for all races, coming.... sometime.


you're getting a cat disease. you dug the grave.

**Pull request closed without merging**